go.mod

```diff
@@ -32,16 +29,14 @@ require (
 	github.com/opencontainers/image-spec v1.0.1
 	github.com/satori/go.uuid v1.2.0 // indirect
 	github.com/sirupsen/logrus v1.6.0
-	github.com/spf13/cobra v0.0.3
-	github.com/spf13/pflag v1.0.3 // indirect
+	github.com/spf13/cobra v1.1.3
 	github.com/yvasiyarov/go-metrics v0.0.0-20140926110328-57bccd1ccd43 // indirect
 	github.com/yvasiyarov/gorelic v0.0.0-20141212073537-a9bba5b9ab50
 	github.com/yvasiyarov/newrelic_platform_go v0.0.0-20140908184405-b21fdbd4370f // indirect
 	golang.org/x/crypto v0.0.0-20200128174031-69ecbb4d6d5d
 	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
-	google.golang.org/api v0.0.0-20160322025152-9bf6e6e569ff
+	google.golang.org/api v0.13.0
 	google.golang.org/cloud v0.0.0-20151119220103-975617b05ea8
```

Comment:
Updating, to v1.1.3 looks to be somewhat problematic.

I suspect the issue is here; this (cloud) dependency is prehistoric, but updating to even the older `v0.1.0` tag breaks our code, due to two refactors;

- https://github.com/googleapis/google-cloud-go/commit/5530fc8457464580b66ade0ebe62b8c63e4530f3
- https://github.com/googleapis/google-cloud-go/commit/6f3b5985ee4df6ee7d93bb7855e7e5ad37a527ae